housekeeping-vacuum-tool/src/main/java/com/hotels/housekeeping/tool/vacuum/manifest/ManifestAttributes.java

```diff
@@ -111,13 +111,11 @@ private URL selectManifestFromJars(ProtectionDomain protectionDomain) throws IOE
     if (jarLocation != null) {
       String containingJar = JAR_PROTOCOL + jarLocation.toString();
       Enumeration<URL> resources = protectionDomain.getClassLoader().getResources(META_INF_MANIFEST_MF);
-      if (resources != null) {
```

Comment:
Since `getResources()` returns an empty enumeration if no resource can be found, the check is not needed.